test/end-to-end/pages/deceased/dod.js

```diff
@@ -1,14 +1,19 @@
 'use strict';
 
 const config = require('config');
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/deceased/dod');
+const commonContentEn = require('app/resources/en/translation/common');
+const commonContentCy = require('app/resources/cy/translation/common');
+const dodContentEn = require('app/resources/en/translation/deceased/dod');
+const dodContentCy = require('app/resources/cy/translation/deceased/dod');
+const pageUnderTest = require('app/steps/ui/deceased/dod');
 
```

Comment:
This is undoing performance work - including these files takes time - do we need to?